surrealdb/clients/http.py

```diff
@@ -155,16 +255,51 @@ async def replace_one(self, table: str, id: str, data: Any) -> Dict[str, Any]:
         return response.result[0]
 
     async def upsert_one(self, table: str, id: str, data: Any) -> Dict[str, Any]:
+        """Upserts a single item in a table.
+
+        This method requires only the fields you wish to be updated,
+        assuming that this id exists. If it doesn't exist, it will be created
+        with the data.
+
+        Parameters
+        ----------
+        table: :class:`str`
+            The table the item is in.
+        id: :class:`str`
+            The id of the item to upsert.
+        data: :class:`Any`
+            The data to upsers the original item with.
+
+        Returns
+        -------
+        :class:`Dict[str, Any]`
+            The new item.
+        """
         response = await self._request(
             method="PATCH",
             uri=f"/key/{table}/{id}",
             data=jsonlib.dumps(data),
         )
-
```

Comment:
why are we removing these whitespaces? it separates out the logic and removing them is harder to read